src/coreComponents/meshUtilities/InternalMeshGenerator.cpp

```diff
@@ -363,8 +285,13 @@ void InternalMeshGenerator::generateMesh( DomainPartition & domain )
   SortedArray< localIndex > & zposNodes = nodeSets.registerWrapper< SortedArray< localIndex > >( string( "zpos" ) ).reference();
   SortedArray< localIndex > & allNodes  = nodeSets.registerWrapper< SortedArray< localIndex > >( string( "all" ) ).reference();
 
+  // Nodesets for applying boundary conditions to a borehole problem
+  SortedArray< localIndex > & rnegNodes = nodeSets.registerWrapper< SortedArray< localIndex > >( string( "rneg" ) ).reference();
+  SortedArray< localIndex > & tnegNodes = nodeSets.registerWrapper< SortedArray< localIndex > >( string( "tneg" ) ).reference();
+  SortedArray< localIndex > & tposNodes = nodeSets.registerWrapper< SortedArray< localIndex > >( string( "tpos" ) ).reference();
+  SortedArray< localIndex > & rposNodes = nodeSets.registerWrapper< SortedArray< localIndex > >( string( "rpos" ) ).reference();
```

Comment:
Do these variables `rnegNodes` ... `rposNodes` make sense in strict cartesian grids?
Same, do `xnegNodes` ... `zposNodes`, `allNodes` all make sense for radial meshes?

If not, it could be possible to have a clean little class hierarchy to keep tighter scope on all of this.
We can also say that we don't care.

Reply:
rnegNodes, rposNodes, etc are now moved to the wellbore mesh class.